### PR TITLE
various updates

### PR DESCRIPTION
**Pull request by hebus**
* when schematic is applied, the application use now the whole json configuration
* image can be sized using the updated image-selector component
* application title can be set using the global configurator

### CODE REVIEW

File: projects/vanilla/src/app/home/home.component.ts [hunk 1]
```diff
@@ -14,15 +13,14 @@ export class HomeComponent implements OnInit {
...
-    private intlService: IntlService
+    private globalService: GlobalService
   ) {}
 
   /**
    * On initialization set the page title
    */
   ngOnInit() {
-    this.titleService.setTitle(this.intlService.formatMessage("msg#app.name"));
+    this.globalService.setTitle();
```
Comment (ericleib) [inline projects/vanilla/src/app/home/home.component.ts]: This logic maybe needs to be in the app.component.ts, so the title is also set in the search component ?

Comment (ericleib) [reply projects/vanilla/src/app/home/home.component.ts]: (or in the app config service)

**Review by ericleib — APPROVED** [review]

File: projects/vanilla/src/app/home/home.component.ts (hunk 1 above)
Comment (hebus) [reply projects/vanilla/src/app/home/home.component.ts]: I've setted it first in the config service, but I don't remember why I've moved it here later

Comment (hebus) [reply projects/vanilla/src/app/home/home.component.ts]: I remember... When you are in the "seach component" and navigate back to "home" clicking in the "Sineaqu" logo the Title does not change

Comment (hebus) [reply projects/vanilla/src/app/home/home.component.ts]: solved this with a TitleStrategy service

**Pull request merged**